santosfamilyfoundation/RuinaMain
# [RUIN-312], [RUIN-307] Update JSON converter and HTML strings to use the human readable ids

# Description

Previously, the HTML strings used to generate HTML and PDF versions of the report were not using the new human-readable IDs but instead relying on the old randomly-generated 8-character IDs from the spreadsheet. Additionally, every component was submitting its data using other randomly-generated ids. For consistency purposes, every component has been updated to submit data using the question's human-readable id, and all of the JSON converters have been updated to reference these ids when looking for the corresponding answers to populate the export format. I manually updated all the HTML strings (the templates for the HTML and PDF exports) to use the human-readable IDs, but there are still some old ids because I cannot tell which questions these ids correspond to.

# Testing
1. Start a new report.
2. Add 2 vehicles and one non-motorist.
3. Populate each of the forms. Make sure to answer a number of questions. Try filling out one text field, one number button selector, one multi-button selector, one single select dropdown, and one multi-select dropdown in each of the forms.
4. Export the form to a desired format. (Try HTML or PDF).
5. Confirm that the export worked correctly.
